lib/execution_engine2/sdk/job_submission_parameters.py

```diff
@@ -58,17 +58,76 @@ def __init__(
         self.memory_MB = _gt_zero(memory_MB, "memory in MB")
         self.disk_GB = _gt_zero(disk_GB, "disk space in GB")
         self.client_group = _check_string(client_group, "client_group")
-        self.client_group_regex = client_group_regex
-        self.as_user = _check_string(as_user, "as_user", optional=True)
-        self.ignore_concurrency_limits = ignore_concurrency_limits
-        sr = scheduler_requirements if scheduler_requirements else {}
+        self.client_group_regex = (
+            None if client_group_regex is None else bool(client_group_regex)
+        )
+        self.bill_to_user = _check_string(bill_to_user, "bill_to_user", optional=True)
```

Comment:
I'm guessing you have already, but this is just a reminder to see if anything else tries to invoke `as_user` as an attribute of the JobRequirements

Reply:
I don't think it does on the develop branch.

Reply:
This is all new code and doesn't interface with anything else yet, so we should be good. Also, given it's new code, it has 100% test coverage (or at least it should, I haven't gotten around to hooking up codecov yet) so that sort of thing should be caught